nest/02.introduction.md

```diff
@@ -0,0 +1,57 @@
+# 1 Introduction
+## 1.1 What is a package manager?
+A package is a compressed file archive containing all of the files coming with a particular software or application.
+
+A package manager allows the user to install, upgrade or remove software packages on their OS, in other words, to **manage the packages**.
+It is similar to an app store for any smartphone, or the Windows Store for example. One of the main differences between those and Nest (and most package managers), is that the upgrading must be manually commanded.
+On a smartphone, the applications are usually automatically upgraded overnight when you're charging your phone, if an update is available.
```

Comment:
I think this actually true.

Reply:
Did you meant to say `isn't true`, or were you just agreeing with what I wrote?

Reply:
yeah sorry, I meant untrue.

Reply:
maybe I shouldn't have based myself on my experience since I've only had Android smartphones. I'll remove that sentence then

Reply:
Idk, I've got an android too and it seems to want to update whenever I have a stable and free internet connection. I think something like this would be ok:
```suggestion
On a smartphone, the applications are usually automatically upgraded.
```